cmd/helpers/config/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package config
+
+import (
+	. "github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+)
+
+var config Sqlconfig
+
+func Update(
+	id string,
+	imageName string,
+	portNumber int,
+	username string,
+	password string,
+	contextName string,
+) {
+	if id == "" {
+		panic("id must be provided")
+	}
+	if imageName == "" {
+		panic("imageName must be provided")
+	}
+	if portNumber == 0 {
+		panic("portNumber must be non-zero")
+	}
+	if password == "" {
+		panic("password must be provided")
+	}
+	if contextName == "" {
+		panic("contextName must be provided")
+	}
+
+	contextName = FindUniqueContextName(contextName, username)
+	endPointName := FindUniqueEndpointName(contextName)
+	userName := username + "@" + contextName
+
+	config.ApiVersion = "v1"
+	config.Kind = "Config"
+	config.CurrentContext = contextName
+
+	config.Endpoints = append(config.Endpoints, Endpoint{
```

Comment:
>append [](http://example.com/codeflow?start=20&length=6)

this says "append" but the function name is "Update".
Shouldn't the function also be Append?

Reply:
Agree, good feedback, renamed to "AddContextWithContainer"